reqwest-retry/src/middleware.rs

```diff
@@ -138,7 +162,8 @@ where
                             .to_std()
                             .map_err(Error::middleware)?;
                         // Sleep the requested amount before we try again.
-                        tracing::warn!(
+                        log_retry!(
+                            self.retry_log_level,
```

Comment:
simpler to use `tracing::event!(self.retry_log_level` here than to make a whole macro for it imo
https://docs.rs/tracing/latest/tracing/macro.event.html

Reply:
I tried to do this initially, but the log level needs to be `'static'`, see this [issue](https://github.com/tokio-rs/tracing/issues/2730) in the tracing crate. I updated the macro in here based on the suggestion in the issue. I've seen other crates with configurable log levels use this kind of macro as well, but let me know if you have an other idea here.

Reply:
Ah, fair. That makes sense then :(
Mind adding a link to that issue in the code from the macro?

Reply:
Sure, added a comment with the issue link to the macro. Also updated the changelog since there was a release today.

Reply:
thank you!